scripts/install_font.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+set -e
+
+# Configure font folder
+FONT_DIR="${HOME}/.local/share/fonts"
+mkdir -p ${FONT_DIR}
+cd ${FONT_DIR}
+
+function Download_Font {
+    if [ $# -eq 1 ]
+    then
+        mkdir ${1}
+        cd ${1}
+
+        wget -qO "${1}.zip" https://www.fontsquirrel.com/fonts/download/${1}
+        unzip ${1}.zip
+        rm -rf ${1}.zip
+
+        cd ..
+    fi
```

Comment:
Is it possible to add an "else echo "using cached font $1", so that we know that the case was used ?